spec.bs

```diff
@@ -527,7 +528,8 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
         1. [=Increment component ad k-anonymity count=] given |adComponentDescriptor|'s
           [=ad descriptor/url=].
   1. Run [=interest group update=] with |auctionConfig|'s [=auction config/interest group buyers=].
-  1. TODO: Update bidCount and prevWinsMs for interest groups that participated in the auction.
+  1. Evaluate [=update bid counts=] with |bidIgs|.
+  1. Evaluate [=update previous wins=] with |winner|.

```

Comment:
(This comment applies to this line, the line above, and my _comment_ above (for the last invocation of "update bid counts").

Can this just be "Run" instead of "Evaluate"?